discord/guild.py

```diff
@@ -3833,3 +3835,93 @@ async def change_voice_state(
         ws = self._state._get_websocket(self.id)
         channel_id = channel.id if channel else None
         await ws.voice_state(self.id, channel_id, self_mute, self_deaf)
+
+    async def fetch_automod_rule(self, automod_rule_id: int, /) -> AutoModRule:
+        """|coro|
+
+        This method will fetch an active automod rule from the guild.
```

Comment:
```suggestion
        Fetch an active automod rule from the guild.
```

Needs versionadded, permissions required, and raises section.